File: src/test/java/com/matag/language/PluralTest.java
package com.matag.language;

import static com.matag.language.Plural.plural;
import static org.assertj.core.api.Assertions.assertThat;

import org.junit.Test;

public class PluralTest {
    @Test
    public void pluralOfNormalWord() {
        assertThat(plural("WIZARD")).isEqualTo("WIZARDS");
    }

    @Test
    public void pluralOfWordsThatNeedEs() {
        assertThat(plural("JELLYFISH")).isEqualTo("JELLYFISHES");
    }

    @Test
    public void pluralOfWordEndingWithF() {
        assertThat(plural("ELF")).isEqualTo("ELVES");
    }

    @Test
    public void pluralOfIrregularWords() {
        assertThat(plural("CYCLOPS")).isEqualTo("CYCLOPES");
    }
}

File: src/test/kotlin/com/matag/language/PluralTest.kt
package com.matag.language

import com.matag.language.Plural.plural
import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test

class PluralTest {
    @Test
    fun pluralOfNormalWord() {
        Assertions.assertThat(plural("WIZARD")).isEqualTo("WIZARDS")
    }

    @Test
    fun pluralOfWordsThatNeedEs() {
        Assertions.assertThat(plural("JELLYFISH")).isEqualTo("JELLYFISHES")
    }

    @Test
    fun pluralOfWordEndingWithF() {
        Assertions.assertThat(plural("ELF")).isEqualTo("ELVES")
    }

    @Test
    fun pluralOfIrregularWords() {
        Assertions.assertThat(plural("CYCLOPS")).isEqualTo("CYCLOPES")
    }
}